pkgs/tools/graphics/mangohud/default.nix

```diff
@@ -186,7 +186,6 @@ stdenv.mkDerivation (finalAttrs: {
     # Only the headers are used from these packages
     # The corresponding libraries are loaded at runtime from the app's runpath
     ++ lib.optional x11Support libX11
-    ++ lib.optional waylandSupport wayland
```

Comment:
Should we set strictDeps?

Reply:
Good point! I also found the issues with the nvidia were actually because of bad buildInputs vs nativeBuildInputs, we might even be able to fix that! testing now.